### DISCUSSION TITLE
Want to delete width attribute on table, td etc

### DISCUSSION THREAD

**Discussion by suchiitsme** [GrapesJS/grapesjs#4204] [Q&A]
When we set width to a table element on StyleManager , its updating both style="width:<value>px" and width="<value>px". On import, On save too this is happening, even if the width attribute is not there, it is adding. I would like to remove the 'width' as attribute and keep only style="width:" property. 
Tried with: 
const attr = model.getAttributes();
	delete attr.width;
But width again gets added when we save it using .runCommand('gjs-get-inlined-html').

Please let me know, how can I achieve this?

**Comment by GedMarc** [comment]
width itself is an invalid attribute on the table element - putting it there has no impact on the rendering what-so-ever

https://www.w3schools.com/TAGS/att_width.asp

**Reply by suchiitsme** [reply]
But it affects in few old browsers. So I want to remove it off.
